drivers/can/can_mcan.h

```diff
@@ -605,19 +1133,64 @@ struct can_mcan_config {
 	const void *custom;
 };
 
+/**
+ * @brief Get an array containing the number of elements in Bosch M_CAN Message RAM
+ *
+ * The order of the array entries is given by the @ref CAN_MCAN_MRAM_CFG definitions.
+ *
+ * @param node_id node identifier
+ * @return array of number of elements
+ */
+#define CAN_MCAN_DT_MRAM_ELEMENTS_GET(node_id)                                                     \
+	{                                                                                          \
+		0, /* offset cell */                                                               \
+			CAN_MCAN_DT_MRAM_STD_FILTER_ELEMENTS(node_id),                             \
+			CAN_MCAN_DT_MRAM_EXT_FILTER_ELEMENTS(node_id),                             \
+			CAN_MCAN_DT_MRAM_RX_FIFO0_ELEMENTS(node_id),                               \
+			CAN_MCAN_DT_MRAM_RX_FIFO1_ELEMENTS(node_id),                               \
+			CAN_MCAN_DT_MRAM_RX_BUFFER_ELEMENTS(node_id),                              \
+			CAN_MCAN_DT_MRAM_TX_EVENT_FIFO_ELEMENTS(node_id),                          \
+			CAN_MCAN_DT_MRAM_TX_BUFFER_ELEMENTS(node_id)                               \
+	}
+
+/**
+ * @brief Get an array containing the base offsets for element in Bosch M_CAN Message RAM
+ *
+ * The order of the array entries is given by the @ref CAN_MCAN_MRAM_CFG definitions.
+ *
+ * @param node_id node identifier
+ * @return array of base offsets for elements
+ */
+#define CAN_MCAN_DT_MRAM_OFFSETS_GET(node_id)                                                      \
+	{                                                                                          \
+		0, /* offset cell */                                                               \
+			CAN_MCAN_DT_MRAM_STD_FILTER_OFFSET(node_id),                               \
+			CAN_MCAN_DT_MRAM_EXT_FILTER_OFFSET(node_id),                               \
+			CAN_MCAN_DT_MRAM_RX_FIFO0_OFFSET(node_id),                                 \
+			CAN_MCAN_DT_MRAM_RX_FIFO1_OFFSET(node_id),                                 \
+			CAN_MCAN_DT_MRAM_RX_BUFFER_OFFSET(node_id),                                \
+			CAN_MCAN_DT_MRAM_TX_EVENT_FIFO_OFFSET(node_id),                            \
+			CAN_MCAN_DT_MRAM_TX_BUFFER_OFFSET(node_id)                                 \
+	}
+
 /**
  * @brief Static initializer for @p can_mcan_config struct
  *
  * @param node_id Devicetree node identifier
  * @param _custom Pointer to custom driver frontend configuration structure
- * @param _read_reg Driver frontend Bosch M_CAN register read function
- * @param _write_reg Driver frontend Bosch M_CAN register write function
+ * @param _ops Pointer to front-end @a can_mcan_ops
+ * @param _ops Pointer to front-end @a can_mcan_callbacks
```

Comment:
Typo `_ops` -> `_cbs`
(Same for `CAN_MCAN_DT_CONFIG_INST_GET`)

Reply:
Thanks, fixed.